omise-woocommerce.php

```diff
@@ -98,6 +93,20 @@ public function init_error_messages(){
 		<?php
 	}
 
+	/**
+	 * @since 3.11
+	 */
+	public function payment_methods() {
```

Comment:
Would this be simpler implemented as a public variable (or constant - depending on the PHP version)?

Reply:
In the project we can call a function `Omise()` to retrieve an instance of Omise object as in the code `Omise()-> payment_methods()`. 👈  I think this one is simpler already. What do you think?

Reply:
I think `Omise()->payment_methods` is simpler. I don't see a need for it to be a function

Reply:
I see. I just pushed the update. Can you review again?